webapp/channels/src/sass/components/_modal.scss

```diff
@@ -242,77 +240,68 @@
         display: flex;
-        min-height: 56px;
-        padding: 14px 48px 11px 15px;
-        border-bottom: 1px solid $light-gray;
-        background: #333;
-        border-radius: 0;
-        border-radius: 3px 3px 0 0;
-        color: $white;
-
-        @include clearfix;
-
-        .modal-title {
-            overflow: hidden;
-            max-width: calc(100% - 80px);
-            flex: 1;
-            color: var(--sidebar-header-text-color);
-            font-size: 17px;
-            line-height: 27px;
-            text-overflow: ellipsis;
-            white-space: nowrap;
-
-            .name {
-                font-weight: 600;
-            }
+        min-height: 76px;
+        align-items: center;
+        justify-content: space-between;
+        padding: 16px 64px 16px 32px;
+        border: none;
+        background: transparent;
 
-            .shared-channel-icon {
-                width: 16px;
-                height: 16px;
-                margin: 0 8px;
-                color: #3d3c40;
-                font-size: 16px;
-                line-height: 16px;
+        &.divider {
+            border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.12);
+        }
 
-                &::before {
-                    margin: 0;
-                }
-            }
+        &::before,
+        &::after {
+            content: none;
         }
 
-        .modal-action {
-            padding: 0;
-            margin: 0;
+        .btn {
+            position: relative;
+            top: -2px;
         }
 
         .close {
             position: absolute;
-            z-index: 5;
-            right: 10px;
-            width: 30px;
-            height: 30px;
-            color: white;
-            font-size: 24px;
-            line-height: 30px;
-            opacity: 0.7;
-            text-shadow: none;
-            transition: all 0.1s ease-in;
+            top: 18px;
+            right: 18px;
+            width: 40px;
+            height: 40px;
+            border-radius: 4px;
+            color: rgba(var(--center-channel-color-rgb), 0.56);
+            font-size: 30px;
+            font-weight: 400;
+            opacity: 1;
+
+            &.icon-close {
+                font-size: 24px;
+            }
 
             &:hover {
-                opacity: 1;
-
-                @include alpha-property(background, $black, 0.1);
+                background: rgba(var(--center-channel-color-rgb), 0.08);
+                color: rgba(var(--center-channel-color-rgb), 0.72);
             }
 
-            .sr-only {
-                display: none;
+            &:active {
+                background: rgba(var(--button-bg-rgb), 0.08);
+                color: v(button-bg);
             }
 
-            span {
-                font-family: 'Open Sans', sans-serif;
-                line-height: 10px;
+            .modal-title {
+                width: 100%;
+                background: transparent;
+                color: v(center-channel-color);
```

Comment:
What is the difference between using this `v` vs `var`

Reply:
v is just a shortcut mixin.

Reply:
ah I see, i wish there was intelli-sence to reference these mixins back to the original definition in the editor